compiler/passes/resolveIntents.cpp

```diff
@@ -330,21 +343,19 @@ IntentTag concreteIntentForArg(ArgSymbol* arg) {
       // No need to warn if the argument intent was going to be converted to
       // `const in`
       if (constIntentForType(arg->type) == INTENT_CONST_REF) {
-        // Want this to be an unstable warning, and not trigger for internal or
-        // standard modules
-        auto mod = fn->getModule();
-        bool shouldWarnInternal = (mod->modTag == MOD_INTERNAL &&
-                                   fWarnUnstableInternal);
-        bool shouldWarnStandard = (mod->modTag == MOD_STANDARD &&
-                                   fWarnUnstableStandard);
-        if (fWarnUnstable && (shouldWarnInternal || shouldWarnStandard ||
-                              mod->modTag == MOD_USER) &&
-            !fNoConstArgChecks) {
-          // Lydia 11/29/23 TODO: use `ArgSymbol->originalIntent` and check this
-          // in a later pass that does more AST transformation.  Potentially
-          // callDestructors?
-          warnForConstIntent(arg);
-        }
+        // Lydia 11/29/23 TODO: use `ArgSymbol->originalIntent` and check this
+        // in a later pass that does more AST transformation.  Potentially
+        // callDestructors?
```

Comment:
Maybe move this TODO comment to the top of the body of warnForInferredConstRef ?

Reply:
I thought about that, but the arg intent is checked here, not in the function.  I could move it to the top of these if statements, though

Reply:
Sure; either way works for me. IMO the comment is about moving the whole thing to a later pass, so could be on the function just as well.